cypress/integration/EventWorkingLists/EventWorkingListsDev/index.js

```diff
@@ -0,0 +1,307 @@
+beforeEach(() => {
+    cy.loginThroughForm();
+});
+
+Given('you open the main page with Ngelehun and malaria case context', () => {
+    cy.server();
+    cy.route('GET', '**/events**').as('getDefaultEvents');
+
+    cy.visit('#/programId=VBqh0ynB2wv&orgUnitId=DiszpKrYNg8');
+});
+
+Then('events should be retrieved from the api using the default query args', () => {
+    cy.wait('@getDefaultEvents', { timeout: 40000 }).as('result');
+
+    cy.get('@result')
+        .its('status')
+        .should('equal', 200);
+
+    cy.get('@result')
+        .its('url')
+        .should('include', 'pageSize=15');
+
+    cy.get('@result')
+        .its('url')
+        .should('include', 'page=1');
+
+    cy.get('@result').its('response.body.events').as('events');
+});
+
+Then('the list should display the events retrieved from the api', () => {
+    cy.get('@events')
+        .then((events) => {
+            cy.get('[data-test="event-working-lists"]')
+                .find('tr')
+                .should('have.length', events.length + 1);
+        });
+
+    cy.get('@events')
+        .then((teis) => {
+            cy.get('[data-test="event-working-lists"]')
+                .find('tr')
+                .each(($teiRow, index) => {
+                    const rowId = $teiRow.get(0).getAttribute('id');
```

Comment:
curious if we could use `data-test` here. 
Perhaps something like
```suggestion
                    const rowId = $teiRow.get(0).getAttribute('data-test');
```

Reply:
https://github.com/dhis2/capture-app/pull/1442#discussion_r560803928